documentation/BrowserChannel.md

```diff
@@ -24,39 +24,45 @@ Also, this component create notifications for every connected client at any time
 This small and documented client illustrates the workflow.
 
 ```javascript
-var io = require("socket.io-client");
+var io = require("ws");
 
 // The client starts the communication
 //
-var socket = io.connect("http://localhost:8081/browserChannel");
+var socket = new ws("ws://localhost:8081/browserChannel");
 
 // When the connection is done, the client tells to the flow manager its id
-//
-socket.on("connect", function () {
-    console.log("## Socket connected");
-    socket.send("org.chrome.cloud4chrome");
-});
 
-// Right after sending the id to the flow manager, the server will return back
-// the current settings in the system (if any)
-//
-socket.on("connectionSucceeded", function (settings) {
-    console.log("## Received the following settings: " + JSON.stringify(settings));
+socket.on("open", function () {
+    console.log("## Socket connected");
+    socket.send(JSON.stringify({
+        type: "connect",
```

Comment:
I couldn't see any reference to the message's "type" when receiving a request neither in the browserChannel nor in the webSocketsSettingsHandler. Is this something you want to add in a near future? The point is that the type isn't being checked/handled at all and we're encouraging the people to use that pattern through this documentation.